datarepo-clienttests/src/main/java/testscripts/KubeConfig.java

```diff
@@ -0,0 +1,177 @@
+package testscripts;
+
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.api.ResourcesApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.*;
+import io.kubernetes.client.openapi.models.V1Deployment;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import utils.DataRepoUtils;
+import utils.FileUtils;
+import utils.KubernetesClientUtils;
+
+public class KubeConfig extends runner.TestScript {
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public KubeConfig() {
+    super();
+  }
+
+  private int filesToLoad;
+  private String datasetCreator;
+  private BillingProfileModel billingProfileModel;
+  private DatasetSummaryModel datasetSummaryModel;
+
+  public void setParameters(List<String> parameters) throws Exception {
+    if (parameters == null || parameters.size() == 0) {
+      throw new IllegalArgumentException(
+          "Must provide a number of files to load between 1 and 25 in the parameters list");
```

Comment:
Nit: Validate for range?

Reply:
After Dan's review, I changed it so we can accept any number of files. But, I forgot to change this message!